db/migrate/20240426144141_add_missing_unique_indexes.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# migration to add missing unique indexes
+class AddMissingUniqueIndexes < ActiveRecord::Migration[7.1]
+  def up
```

Comment:
**nitpick (blocking):** This can all be out in a `change` method instead of having `up` and `down` as rails knows how to handle down for this.

Reply:
Updated in 308737e64548a6b763fd336af4bbc86b2eabb9c4